apps/challenges/models.py

```diff
@@ -177,6 +179,11 @@ def is_active(self):
     sender=Challenge,
     weak=False,
 )
+signals.post_save.connect(
+    model_field_name(field_name="approved_by_admin")(challenge_start_notifier),
+    sender=Challenge,
+    weak=False,
+)
 signals.post_save.connect(
     model_field_name(field_name="approved_by_admin")(create_eks_cluster),
```

Comment:
@KhalidRmb We need to add a condition here that the eks cluster should only be created when the challenge is docker based.
cc: @Ayukha